src/migrate/helpers.ts

```diff
@@ -10,7 +10,7 @@ export const createChunkCounter = async (
   chunkSize: number
 ): Promise<number[]> => {
   const MigrationDataElement = await createMigrationDataElements(sequelize);
-  const where = { migrationId, isProcessed: false };
+  const where = { migrationId, migratedAt: null };
```

Comment:
Business Logic is that this worker is only taking things that are NOT processed, while Failure Worker takes whats NOT migrates but IS processed